msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/entity/MessageForumsEntityContentProducer.java

```diff
@@ -54,6 +56,8 @@
 	// runtime dependency
 	private List removeEvents = null;
 	
+	@Setter private FormattedText formattedText;
```

Comment:
Why not go the whole hog and remove the setters for those other private vars in there? Just move @Setter to the class definition and strip those setters out. You may as well.

Reply:
OK